locales/en/email-strings.ftl

```diff
@@ -90,3 +90,47 @@ email-breach-alert-all-data-points-title = Your exposed data:
 email-breach-alert-all-next-steps-lead = We’ll guide you step-by-step on how to resolve this data breach.
 email-breach-alert-all-next-steps-cta-label = Let’s get started
 email-breach-alert-all-next-steps-button-dashboard = Go to Dashboard
+
+# Monthly Report Free User
+
+email-monthly-report-hero-free-heading = Your monthly { -brand-mozilla-monitor } report
+email-monthly-report-hero-free-body = A summary of what needs your attention, and what’s been resolved over the past month.
+# Variables:
+#   $data_point_count (number) - The number of data breaches that were exposed
```

Comment:
```suggestion
# This string is displayed under a large numeral that indicates the total number
# of data breaches that have exposed the user’s information. Don’t add $data_point_count to
# your localization, because it would result in the number showing twice.
# Variables:
#   $data_point_count (number) - The number of data breaches that were exposed
```

Still missing the comment for all the strings. You can also create it as a group level comment, and reset if after the group of strings.